applications/services/gui/view_dispatcher.h

```diff
@@ -97,6 +106,10 @@ void view_dispatcher_set_navigation_event_callback(
 
 /** Set tick event handler
  *
+ * @warning Requires the event loop to be owned by the view dispatcher, i.e.
```

Comment:
Applications use `view_dispatcher_set_tick_event_callback` as `furi_event_loop_tick_set` because prior to `view_dispatcher_alloc_ex` and `view_dispatcher_get_event_loop` there was no way to access the event loop directly. I want applications to be aware of the fact that if they create a view dispatcher via `_alloc_ex`, they're the ones managing the event loop, and hence they must use the direct event loop API instead instead of the indirect view dispatcher API.

Another way to do this would be to deprecate `view_dispatcher_set_tick_event_callback` and ask applications to set their tick callbacks directly after getting the loop via `view_dispatcher_get_event_loop`.